docs/python_docs/python/tutorials/packages/ndarray/index.rst

```diff
@@ -45,10 +45,16 @@ NDArray
 
       For Sparse NDArray tutorials
 
+   .. card::
+      :title: NP on MXNet reference
+      :link: deepnumpy/index.html
+
+      This section contains the mxnet.np API reference documentation
 
 .. toctree::
    :hidden:
    :glob:
 
    *
-   sparse/index
\ No newline at end of file
+   sparse/index
+   deepnumpy/index
```

Comment:
this should be at top level, not under ndarray.

Reply:
Will move this part to top level